resources/lib/playerinfo.py

```diff
@@ -193,7 +193,10 @@ def update_position(self):
     def update_total(self):
         ''' Update the total video time '''
         try:
-            self.total = self.getTotalTime()
+            total = self.getTotalTime()
+            # Kodi Player sometimes returns a total time of 0.0 and this causes unwanted behaviour with VRT NU Resumepoints API.
+            if total > 0.0:
```

Comment:
Good you are anticipating negative total times here too. Who know what will happen in the future ;-)